judge/models/contest.py

```diff
@@ -54,6 +54,11 @@ class Meta:
 
 
 class Contest(models.Model):
+    SCOREBOARD = (
+        ('V', _('Visible')),
```

Comment:
Would suggest that you create constants like `SCOREBOARD_VISIBLE` on the `Contest` to avoid future typos.